docs/website/mkdocs.yml

```diff
@@ -80,7 +80,11 @@ markdown_extensions:
       options:
         custom_icons:
           - overrides/.icons
-  - pymdownx.superfences
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_code_format
```

Comment:
could split this into a separate PR, and https://squidfunk.github.io/mkdocs-material/reference/diagrams/ requires insiders (which we haven't set up yet)

Reply:
Removed.